etc/schema.json

```diff
@@ -4048,15 +4048,27 @@
                                 "type": "string"
                             }
                         },
-                        "time": {
-                            "type": "string",
-                            "optional": true,
-                            "description": "Start and stop times for a session"
-                        },
-                        "repeat_time": {
```

Comment:
`repeat_time` is not in 7, right ?
This is a breaking change, but it is ok since it is only in master/8

Reply:
Right, it's not in 7.

Reply:
ok for me